### PR TITLE
Fix auto merge conflict 154

### PR DESCRIPTION
**Pull request by nvliyuan**
Fix auto-merge conflict 154

### CODE REVIEW

File: examples/Spark-cuSpatial/gpu-run.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ rm -rf $DATA_OUT_PATH
 # the path to keep the jars of cudf, spark-rapids & spark-cuspatial
 JARS=$ROOT_PATH/jars
 
-JARS_PATH=$JARS/cudf-22.02.0-cuda11.jar,$JARS/rapids-4-spark_2.12-22.02.0.jar,$JARS/spark-cuspatial-22.06.0-SNAPSHOT.jar
+JARS_PATH=$JARS/cudf-22.04.0-cuda11.jar,$JARS/rapids-4-spark_2.12-22.04.0.jar,$JARS/spark-cuspatial-22.06-SNAPSHOT.jar
```
Comment (GaryShen2008) [inline examples/Spark-cuSpatial/gpu-run.sh:34]: why we change 22.06.0 to 22.06?

File: examples/Spark-cuSpatial/notebooks/spatial_join_standalone.ipynb [hunk 2]
```diff
@@ -9,7 +9,7 @@
    "source": [
     "from pyspark.sql import SparkSession\n",
     "spark = SparkSession.builder \\\n",
-    "    .config(\"spark.jars\", \"/data/cuspatial_data/jars/cudf-22.02.0-cuda11.jar,/data/cuspatial_data/jars/rapids-4-spark_2.12-22.02.0.jar,/data/cuspatial_data/jars/spark-cuspatial-22.06.0-SNAPSHOT.jar\") \\\n",
+    "    .config(\"spark.jars\", \"/data/cuspatial_data/jars/cudf-22.04.0-cuda11.jar,/data/cuspatial_data/jars/rapids-4-spark_2.12-22.04.0.jar,/data/cuspatial_data/jars/spark-cuspatial-22.06-SNAPSHOT.jar\") \\\n",
```
Comment (GaryShen2008) [inline examples/Spark-cuSpatial/notebooks/spatial_join_standalone.ipynb:12]: Same problem

Comment (nvliyuan) [reply examples/Spark-cuSpatial/notebooks/spatial_join_standalone.ipynb]: updated

File: examples/Spark-cuSpatial/gpu-run.sh (hunk 1 above)
Comment (nvliyuan) [reply examples/Spark-cuSpatial/gpu-run.sh]: updated

**Review by GaryShen2008 — APPROVED** [review]

**Pull request merged**